scripts/combine/setupCombine.py

```diff
@@ -132,6 +135,9 @@ def make_parser(parser=None):
     parser.add_argument("--pseudoDataAxes", type=str, nargs="+", default=[None], help="Variation axes to use as pseudodata for each of the histograms")
     parser.add_argument("--pseudoDataIdxs", type=str, nargs="+", default=[None], help="Variation indices to use as pseudodata for each of the histograms")
     parser.add_argument("--pseudoDataFile", type=str, help="Input file for pseudodata (if it should be read from a different file)", default=None)
+    parser.add_argument("--pseudoDataFitInputFile", type=str, help="Input file for pseudodata (if it should be read from a fit input file)", default=None)
+    parser.add_argument("--pseudoDataFitInputChannel", type=str, help="Input chnnel name for pseudodata (if it should be read from a fit input file)", default="ch0")
+    parser.add_argument("--pseudoDataFitInputDownUp", type=str, help="DownUp variation for pseudodata (if it should be read from a fit input file)", default="Up")
```

Comment:
You could restrict the possibilities by specifying choices=["Up","Down"] or use it as a boolean argument. Only if other updates are done

Reply:
Yes restricting the choices makes sense

Reply:
Do you want to make this change? I'm in either case fine with merging

Reply:
I think we can merge it and add the choice restriction later.

Reply:
One can invent a syntax to give all these details with a single option, like --pseudoDataFitInput "file=xx;chan=yy;upDown=zz", with some simple parsing of keywords file/chan/upDown. Since the number of options is increasing a lot, this might simplify the usage

Reply:
I think this is a bad idea. To make it easy for newcomers (and everyone else) it's best to simplify and restrict the options as much as possible. Having a complicated parsing on top of the actual parsing goes against this

Reply:
ok, I see your point. Then maybe we should at least make sure than some of these options cannot be called unless others are also called, since a dependency is assumed (referring to the main pseudodata options now). Similarly to what is done for unfolding/theoryagnostic and specific options like --noiAspoi. 
It can be left for another time though.